nvflare/private/fed/utils/fed_utils.py

```diff
@@ -236,3 +240,34 @@ def create_stats_pool_files_for_job(workspace: Workspace, job_id: str, prefix=No
         err = f"Failed to create stats pool summary file {summary_file}: {secure_format_exception(e)}"
     StatsPoolManager.close()
     return err
+
+
+def split_gpus(gpus) -> ([str], bool):
```

Comment:
Now since this func returns list of str, please change the signature to -> [str]

Reply:
done